src/libasr/codegen/asr_to_c_cpp.h

```diff
@@ -469,16 +504,18 @@ R"(#include <stdio.h>
                     if (gen_stdcomplex) {
                         sub = "std::complex<float> ";
                     } else {
-                        sub = "float_complex_t ";
```

Comment:
This change should be reverted. See: https://github.com/lcompilers/lpython/pull/2193/files